Firmware/mesh_bed_calibration.cpp

```diff
@@ -2872,14 +2872,14 @@ bool sample_mesh_and_store_reference()
 		}
         mbl.set_z(0, 0, current_position[Z_AXIS]);
     }
-    for (int8_t mesh_point = 1; mesh_point != MESH_MEAS_NUM_X_POINTS * MESH_MEAS_NUM_Y_POINTS; ++ mesh_point) {
+    for (uint8_t mesh_point = 1; mesh_point != MESH_MEAS_NUM_X_POINTS * MESH_MEAS_NUM_Y_POINTS; ++ mesh_point) {
```

Comment:
please add a `static_assert(MESH_MEAS_NUM_X_POINTS * MESH_MEAS_NUM_Y_POINTS < 255, "overflow.....")`

Reply:
done

Reply:
I thought you'd add a reasonable message as well why the static assert fails :wink: 
Something like:
_MESH_MEAS_NUM_X_POINTS * MESH_MEAS_NUM_Y_POINTS > 255: it would overflow the uint8_t range - the following for-cycle would misbehave_